cpp/CMakeLists.txt

```diff
@@ -256,8 +256,8 @@ link_directories("${CMAKE_CUDA_IMPLICIT_LINK_DIRECTORIES}" # CMAKE_CUDA_IMPLICIT
 
 add_library(cudf SHARED
             src/comms/ipc/ipc.cu
-            src/column/column.cpp
-            src/column/context.cpp
+            src/column/legacy/column.cpp
```

Comment:
Wondering if this should go to `src/legacy/column/...` instead? Likewise `src/legacy/bitmask/...`, and `tests/legacy/column/...` etc.  

Group all the legacy stuff together? It's not a huge deal, but it makes it really easy to see in the #include lines that a legacy file is included / a dependency. For tests, it could allow easily enabling / disabling legacy tests if we need to.

Reply:
The headers are all grouped in `cudf/legacy` so it will be obvious when someone is including a file from there. 

The source files are still located in their topical `src/*/legacy` directories, likewise with tests.